server/src/main/java/org/apache/lucene/search/XCombinedFieldQuery.java

```diff
@@ -0,0 +1,466 @@
+/* @notice
```

Comment:
I don't think this should be here?

Reply:
Are you talking about the `@notice` annotation? This is required by our build for non-Elastic licensed code so that the notices are copied: #57017. I based this off other Lucene classes that we've copied like `XMoreLikeThis` and `IndexableBinaryStringTools`.